packages/run-protocol/src/vpool-xyk-amm/constantProduct/calcFees.js

```diff
@@ -32,7 +32,7 @@ const amountGT = (left, right) =>
  * Apply the feeRatio to the amount that has a matching brand. This used to
  * calculate fees in the single pool case.
  *
- * @param {{ amountIn: Amount, amountOut: Amount}} amounts - a record with two
+ * @param {{ amountIn: Amount; amountOut: Amount }} amounts - A record with two
```

Comment:
Why didn't it remove this hyphen?

Reply:
It doesn't do anything with hyphens. I removed some manually for consistency in this package.